pkg/cmd/release-payload-controller/batchjob_controller.go

```diff
@@ -0,0 +1,279 @@
+package release_payload_controller
+
+import (
+	"context"
+	"fmt"
+	"github.com/openshift/release-controller/pkg/apis/release/v1alpha1"
+	releasepayloadclient "github.com/openshift/release-controller/pkg/client/clientset/versioned/typed/release/v1alpha1"
+	releasepayloadinformer "github.com/openshift/release-controller/pkg/client/informers/externalversions/release/v1alpha1"
+	releasepayloadlister "github.com/openshift/release-controller/pkg/client/listers/release/v1alpha1"
+	releasecontroller "github.com/openshift/release-controller/pkg/release-controller"
+	releasepayloadhelpers "github.com/openshift/release-controller/pkg/releasepayload/v1alpha1helpers"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	batchv1informers "k8s.io/client-go/informers/batch/v1"
+	batchv1listers "k8s.io/client-go/listers/batch/v1"
+	"reflect"
+	"time"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const (
+	// ReleaseCreationJobUnknownMessage release creation job unknown message
+	ReleaseCreationJobUnknownMessage = "Release creation job unknown"
+
+	// ReleaseCreationJobPendingMessage release creation job pending message
+	ReleaseCreationJobPendingMessage = "Release creation job pending"
+
+	// ReleaseCreationJobFailureMessage release creation job failure message
+	ReleaseCreationJobFailureMessage = "Release creation Job failed"
+
+	// ReleaseCreationJobSuccessMessage release creation job success message
+	ReleaseCreationJobSuccessMessage = "Release creation Job completed"
+)
+
+// BatchJobStatusController is responsible for watching batchv1.Jobs, in the job-namespace, and
```

Comment:
Is this actually a ReleaseCreationStatusController?

Reply:
Renamed. Done